source/network/ODServer.cpp

```diff
@@ -606,6 +610,7 @@ bool ODServer::processClientNotifications(ODSocketClient* clientSocket)
             const std::string& nick = clientSocket->getPlayer()->getNick();
             int32_t id = clientSocket->getPlayer()->getId();
             int seatId = seat->getId();
+            seat->setMapSize(gameMap->getMapSizeX(), gameMap->getMapSizeY());
```

Comment:
Because in the editor mode, we don't have the ClientNotification::seatConfigurationSet event. So, in editor mode, we do that in the ClientNotification::setNick event (ie last state before game mode).